src/javascript/SelectorTypes/Picker/PickerDialog/RightPanel/ContentLayout/ContentTable/ContentTable.jsx

```diff
@@ -122,6 +125,15 @@ export const ContentTable = ({rows, isContentNotFound, totalCount, isLoading, ca
         }
     }, [rows, isStructured, toggleRowExpanded, firstLoad]);
 
+    // Expand structured view selections when root row is changed
+    const rootRowUuid = useRef();
+    useEffect(() => {
+        if (isStructured && rootRowUuid.current !== rows[0]?.uuid && selection && selection.length) {
+            rootRowUuid.current = rows[0].uuid;
+            expandSelection(selection);
```

Comment:
I was wondering if it's possible to use initialState.expanded in the useTable instead of having an effect ? I'm not sure it's possible as the root row can be updated